scripts/utilities/regeneratePackageLocks/index.js

```diff
@@ -0,0 +1,9 @@
+const removeExistingPackageLock = require('./removeExistingPackageLock');
```

Comment:
I dont think anything in scripts/utilities/regeneratePackageLocks is being used?

Reply:
Yeh you're right but it's been a handy script to run when package's `package-lock.json` files get out of sync and `ci:packages` script refuses to run when this happens.

I could add this to the scripts section in the `package.json` if you think it would get used often.

Reply:
Yeah I think so, better than it being hidden down here :P